tests/testthat/test-01-parallel.R

```diff
@@ -5,7 +5,7 @@ SlowGettingDirectoryStore <- R6::R6Class("SlowGettingDirectoryStore",
   public = list(
     get_item = function(key) {
       # Simulate a slow read such as an HTTP request.
-      Sys.sleep(1.0/10.0)
+      Sys.sleep(1.0/5)
```

Comment:
I think this makes the sleep duration longer, were you intending to make it shorter?

Reply:
I was having a hard time getting the parallel version to be consistently faster due to per process overhead I think.